aerogear/mobile-developer-console
# [WIP] feat: add a nodejs server and refactor app CRUDL endpoints

## Motivation

This will be a long running branch to contain the refactoring work of MDC.

This PR:

* Adds a new nodejs server
* Refactored the apps CRUDL & watch endpoints
* Add script for local development

To verify:

* Start a new minishift server and make sure the mobileclients CRDs & roles are setup correctly
* Run `local_dev.sh`
* In the console, try create, edit, delete apps.
